hicexplorer/hicFindEnrichedContacts.py

```diff
@@ -812,8 +815,8 @@ def main(args=None):
         hic_ma.diagflat()
 
     if args.method == 'obs/exp':
-        hic_ma.convert_to_obs_exp_matrix(maxdepth=args.depth, perchr=args.perchr)
+        hic_ma.convert_to_obs_exp_matrix(maxdepth=args.maxDepth, perchr=args.perchr)
     else:
-        hic_ma.convert_to_zscore_matrix(maxdepth=args.depth, perchr=args.perchr)
+        hic_ma.convert_to_zscore_matrix(maxdepth=args.maxDepth, perchr=args.perchr)
```

Comment:
If the parameter is changed from `maxDepth` to `depth` than please change it in the source code too.

Reply:
it's the other way round.. it was called `--depth` in the argparse but maxDepth everywhere else. so I changed it to maxDepth..

Reply:
You are right, my fault.